elastic/apm-server
# Add `transaction.name` mapping to `logs-apm.error`

## Motivation/summary

Adds the missing mapping for `transaction.name` to `logs-apm.error`.

## Checklist



~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)~
~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/master/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~
~- [ ] Documentation has been updated~

## How to test these changes

1. `docker-compose up -d`
2. `cd systemtest &&  go test -v -run TestErrorIngest .`
3. Navigate to `Kibana > Dev tools`
```javascript
GET logs-apm.error*/_search
{
  "query": {
    "exists": {
      "field": "transaction.name"
    }
  }, 
  "_source": [false], 
  "fields": [
    "transaction.type",
    "transaction.name"
  ]
}
```

Verify that the query returns a result with the `transaction.name` set.

## Related issues

Fixes regression found in #6539
